src/saving/SaveManagerGUI.cs

```diff
@@ -166,6 +177,19 @@ private void DeleteSelectedButtonPressed()
         deleteSelectedConfirmDialog.PopupCenteredMinsize();
     }
 
+    private void DeleteOldButtonPressed()
+    {
+        int autoSavesToDeleteCount = (currentAutoSaveCount - 1).Clamp(0, Settings.Instance.MaxAutoSaves);
+        int quickSavesToDeleteCount = (currentQuickSaveCount - 1).Clamp(0, Settings.Instance.MaxQuickSaves);
+
+        deleteOldConfirmDialog.DialogText =
+            "Deleting all old Auto and Quick saves cannot be undone, " +
+            "are you sure you want to permanently delete the following?\n" +
+            $" >>> {autoSavesToDeleteCount} Auto save(s)\n" +
```

Comment:
I think that using ` - ` instead of ` >>> ` would look more consistent with the rest of the game.